### PR TITLE
Bugfixes, improvements and new symbols/nosymbols option for all styles

### PR DESCRIPTION
**Pull request by cryptointerest**
_Bugfixes, improvements and new symbols/nosymbols option for all styles_

- fix Researcher ID (still) doesn't display Issue #29.
- make options 'symbols' and 'nosymbols' available to all styles to toggle whether 
  symbols should be used or not. Include symbols in oldstyle by default. 
- migrate older newcommand syntax in moderncvcollection.sty to the xparse syntax.
- Fix resumé typo in template. Is now résumé.
- remove loading lmodern fonts in all styles as they can produce issues with 
  german characters, see https://tex.stackexchange.com/questions/496630/lualatex-problems-with-german-characters.
- remove loading inputenc package by default. Only load the package when pdflatex is 
  used.

### CODE REVIEW

**Comment by stephanlachnit** [comment]
Btw you can close issues with PRs automatically if you write `Close #29` in the description of the PR.

Will take a look at it tmw.

**Comment by cryptointerest** [comment]
ah nice, thanks.

File: moderncviconsawesome.sty [hunk 1]
```diff
@@ -45,7 +45,7 @@
 \renewcommand*{\skypesocialsymbol}        {{\small\faSkype}~}
 \renewcommand*{\orcidsocialsymbol}        {{\small\aiOrcid}~}
 \renewcommand*{\researchgatesocialsymbol} {{\small\aiResearchGate}~}
-\renewcommand*{\researchidsocialsymbol}   {{\small\aiResearcherID}~}
+\renewcommand*{\researcheridsocialsymbol}   {{\small\aiResearcherID}~}       % alternative: \aiResearcherIDSquare
```
Comment (stephanlachnit) [inline moderncviconsawesome.sty:48]: Spacing, two spaces too much.

File: moderncviconsletters.sty [hunk 2]
```diff
@@ -50,7 +50,7 @@
 \renewcommand*{\skypesocialsymbol}        {\textbf{sk}~}
 \renewcommand*{\orcidsocialsymbol}        {\textbf{orcid}~}
 \renewcommand*{\researchgatesocialsymbol} {\textbf{rg}~}
-\renewcommand*{\researchidsocialsymbol}   {\textbf{ri}~}
+\renewcommand*{\researchidersocialsymbol}   {\textbf{ri}~}
```
Comment (stephanlachnit) [inline moderncviconsletters.sty:53]: Same here.

**Review by stephanlachnit — COMMENTED** [review]
Cool, thanks. Will merge once the two minor spacings are adjusted.

File: moderncviconsawesome.sty (hunk 1 above)
Comment (cryptointerest) [reply moderncviconsawesome.sty]: done

File: moderncviconsletters.sty (hunk 2 above)
Comment (cryptointerest) [reply moderncviconsletters.sty]: done.

Comment (mefuller) [reply moderncviconsletters.sty]: Appears to be a typo: researcherid vs. researchider

Comment (cryptointerest) [reply moderncviconsletters.sty]: daaaaamn. thanks.

**Review by stephanlachnit — APPROVED** [review]

**Pull request merged**